Resources/config/services.xml

```diff
@@ -5,34 +5,64 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="doctrine_migrations.diff_command" class="Doctrine\Bundle\MigrationsBundle\Command\MigrationsDiffDoctrineCommand">
+        <service id="doctrine_migrations.diff_command" class="Doctrine\Migrations\Tools\Console\Command\DiffCommand">
+            <argument>doctrine:migrations:diff</argument>
+            <argument type="service" id="doctrine.migratrions.di"/>
```

Comment:
Is this a typo?
```suggestion
            <argument type="service" id="doctrine.migrations.di"/>
```
(by the way, same for all other statements with the same reference)